cli/azd/cmd/pipeline.go

```diff
@@ -66,6 +66,11 @@ func (pc *pipelineConfigFlags) Bind(local *pflag.FlagSet, global *internal.Globa
 	// there no customer input using --provider
 	local.StringVar(&pc.PipelineProvider, "provider", "",
 		"The pipeline provider to use (github for Github Actions and azdo for Azure Pipelines).")
+	local.StringVar(&pc.ServiceManagementReference, "smf", "",
```

Comment:
I think I prefer using a full name like `--serviceManagementReference` (or perhaps even `--applicationServiceManagementReference` both here and for the config key name.  I think the verbosity here is fine.

Reply:
+1 to this.  I'd spell out the parameter but include a short name as `smr` (Any reason why you went with `smf` instead?)

Reply:
> Any reason why you went with smf instead

Let's call it a typo! lol.  It should've been `--smr` since the beginning.  I'll use `--applicationServiceManagementReference` 

I wasn't able to use a short name directly with Cobra+pfArgs as short flags must be one character long only.  I think we can use `-m`.